### PR TITLE
Fix ruby for-loops examples

### PR DESCRIPTION
**Pull request by dervish86**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/arabic/ruby/ruby-for-loop/index.md [hunk 1]
```diff
@@ -8,21 +8,21 @@ localeTitle: روبي للحلقات
 
 للحلقات هي مجرد مثال واحد على التكرار أو التكرار على العناصر. في ما يلي مثال على حلقة for:
 
- `for element in array do 
-  puts element 
- end 
-` 
+ `for element in array do
```
[the remark this answers is not in the document]
Comment (dervish86) [reply guide/arabic/ruby/ruby-for-loop/index.md]: Set the same formatting as in other localizations

**Review by PurpleCatfish — APPROVED** [review]

**Comment by RandellDawson** [comment]
@dervish86 I corrected your `array.each do syntax`

**Comment by RandellDawson** [comment]
LGTM

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**